tribe-ext-list-venues-organizers-shortcodes.php

```diff
@@ -97,116 +103,162 @@ protected function format() {
 
 			while ( $this->query->have_posts() ) {
 				$this->query->the_post();
-				$shortcode_type = 'tribe_organizer' == $this->atts['post_type'] ? 'organizer' : 'venue';
-				$id             = get_the_ID();
+				$post_id = get_the_ID();
 
 				// count upcoming events
 				$args = array(
 					'start_date' => date( 'Y-m-d H:i:s' ),
 				);
 
-				if ( 'organizer' == $shortcode_type ) {
-					$args['organizer'] = $id;
-				} else {
-					$args['venue'] = $id;
+				if ( Tribe__Events__Main::ORGANIZER_POST_TYPE == $this->atts['post_type'] ) {
+					$args['organizer'] = $post_id;
+				} elseif ( Tribe__Events__Main::VENUE_POST_TYPE == $this->atts['post_type'] ) {
+					$args['venue'] = $post_id;
 				}
 
+				$args = apply_filters( 'tribe_ext_list_venues_organizers_shortcodes_tribe_get_events_per_post_args', $args, $post_id, $this->atts );
+
 				$events       = tribe_get_events( $args );
 				$events_count = count( $events );
 
 				// generate HTML
-				if ( 'yes' != $hide_empty || ( 'yes' == $hide_empty && $events_count > 0 ) ) :
-
-					$item       = '<li id="' . esc_attr( $this->atts['post_type'] . '-' . $id ) . '" class="tec list ' . $this->atts['post_type'] . '">';
+				if (
+					'yes' != $hide_empty
+					|| (
+						'yes' == $hide_empty
```

Comment:
Formatting here or the next line is off

Reply:
@barryhughes Why is this formatting not correct?

Reply:
We're using an initial indent of tabs on line 128 (correct) but a mix of tabs and spaces on line 129 (incorrect). I could have been clearer first time, though: the _following_ line needs to be fixed.

Reply:
gotcha (I think). published regardless ;)